src/Raven.Server/NotificationCenter/Paging.cs

```diff
@@ -160,5 +149,28 @@ public void Dispose()
         {
             _pagingTimer?.Dispose();
         }
+        internal class PagingInformation
```

Comment:
Do we need to be concerned that this will now allocate an object on each call? Can we make this a struct, instead?